gallery/src/styles/NxCard/NxCardPageExamples.scss

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+// Reveal problems with box-sizing set to border-box, see https://issues.sonatype.org/browse/RSC-84
+
+.nx-card__content--2-column-example {
```

Comment:
Typically our example classes start with `.gallery` to make it clear that they aren't part of the RSC lib

Reply:
> Typically our example classes start with `.gallery` to make it clear that they aren't part of the RSC lib

Right: https://github.com/sonatype/sonatype-react-shared-components/pull/307/commits/17a7663a804fa121012ada6de25f21f4a2a1837c